kclvm/vfs/src/sourcemap.rs

```diff
@@ -0,0 +1,62 @@
+use std::{fs, path::PathBuf, sync::Arc};
+
+use anyhow::Ok;
+use kclvm_span::FilePathMapping;
+use parking_lot::RwLock;
+
+use crate::vfs::VFS;
+
+pub struct SourceFile {
+    sf_inner: Arc<RwLock<kclvm_span::SourceFile>>,
+}
+
+pub struct SourceMapVfs {
+    sm_inner: Arc<RwLock<kclvm_span::SourceMap>>,
+}
+
+impl SourceMapVfs {
+    pub fn new() -> Self {
+        SourceMapVfs {
+            sm_inner: Arc::new(RwLock::new(kclvm_span::SourceMap::new(
+                FilePathMapping::empty(),
+            ))),
+        }
+    }
+}
+
+impl VFS for SourceMapVfs {
```

Comment:
Why do they appear simultaneously e.g., `VFS` and `Vfs`. Considering using the same name `Vfs`.